harmony/harmony.py

```diff
@@ -613,6 +614,9 @@ def status(self, job_id: str) -> dict:
                 'updated_at': updated_at_dt,
                 'created_at_local': created_at_dt.replace(microsecond=0).astimezone().isoformat(),
                 'updated_at_local': updated_at_dt.replace(microsecond=0).astimezone().isoformat(),
+                'data_expiration': data_expiration_dt,
+                'data_expiration_local': data_expiration_dt.
```

Comment:
Nit in this case it might be more readable to put on one long line - just so it looks like `created_at_local` and  `updated_at_local` - if an autoformatter is making it this way though don't fight it just leave it this way.